Source/AhriAndBear/UI/NewspaperWidget.cpp

```diff
@@ -0,0 +1,5 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "NewspaperWidget.h"
+
```

Comment:
We can probably delete this file if it's empty

Reply:
I think we can leave it for later since the current newspaper widget was inherited from it.